packages/expo-calendar/ios/Records/CalendarRecords.swift

```diff
@@ -117,3 +117,22 @@ struct Reminder: Record {
   @Field
   var completionDate: Either<String, Double>?
 }
+
+struct OpenInCalendarOptions: Record {
+  @Field
+  var id: String
+  @Field
+  var instanceStartDate: Either<String, Double>?
+}
+
+struct CalendarResponse: Record {
+  @Field
+  var action: String = "done"
```

Comment:
Can this be made into an enum instead of a raw string?

```swift
  enum ResponseAction: Enumerable {
    case done
    case cancelled
    case deleted
    case responded
 }
```

Reply:
Thanks, I didn't know it's possible. But it actually wasn't supported - waiting for https://github.com/expo/expo/pull/30191/files#diff-7e17df519c28e48eb95ebc98f7102732b22f66dca04279893be7451797fd17a1 now